yarn.lock

```diff
@@ -20844,14 +20826,6 @@ worker-rpc@^0.1.0:
   dependencies:
     microevent.ts "~0.1.1"
 
-wrap-ansi@^3.0.1:
-  version "3.0.1"
-  resolved "https://registry.yarnpkg.com/wrap-ansi/-/wrap-ansi-3.0.1.tgz#288a04d87eda5c286e060dfe8f135ce8d007f8ba"
-  integrity sha1-KIoE2H7aXChuBg3+jxNc6NAH+Lo=
-  dependencies:
-    string-width "^2.1.1"
-    strip-ansi "^4.0.0"
-
```

Comment:
What's the reason for all the `lock` changes?

Reply:
I wish I understood this better. All I know is I'm busily going about my business - not making package changes - and then suddenly I run `yarn` and I get this. So I guess either I've merged in `main` and got some weirdness where the lock wasn't updated from another branch or this is somehow date related?